Makefile

```diff
@@ -3,16 +3,16 @@ JC=javac
 JAR=jar
 KAFKAPATH=./kafka-0.7.2-incubating-src
 HTTPCOMPONENTSPATH=./httpcomponents-client-4.3.6
-CLASSPATH="$(HTTPCOMPONENTSPATH)/lib/commons-codec-1.6.jar:$(HTTPCOMPONENTSPATH)/lib/commons-logging-1.1.3.jar:$(HTTPCOMPONENTSPATH)/lib/fluent-hc-4.3.6.jar:$(HTTPCOMPONENTSPATH)/lib/httpclient-4.3.6.jar:$(HTTPCOMPONENTSPATH)/lib/httpclient-cache-4.3.6.jar:$(HTTPCOMPONENTSPATH)/lib/httpcore-4.3.3.jar:$(HTTPCOMPONENTSPATH)/lib/httpmime-4.3.6.jar:$(KAFKAPATH)/project/boot/scala-2.8.0/lib/scala-compiler.jar:$(KAFKAPATH)/project/boot/scala-2.8.0/lib/scala-library.jar:$(KAFKAPATH)/core/target/scala_2.8.0/kafka-0.7.2.jar:$(KAFKAPATH)/core/lib/*.jar:$(KAFKAPATH)/perf/target/scala_2.8.0/kafka-perf-0.7.2.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/jopt-simple-3.2.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/log4j-1.2.15.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/snappy-java-1.0.4.1.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/zkclient-0.1.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/zookeeper-3.3.4.jar:./jewelcli-0.8.7.jar:./kafkaSpraynozzle.jar"
+CLASSPATH="$(HTTPCOMPONENTSPATH)/lib/commons-codec-1.6.jar:$(HTTPCOMPONENTSPATH)/lib/commons-logging-1.1.3.jar:$(HTTPCOMPONENTSPATH)/lib/fluent-hc-4.3.6.jar:$(HTTPCOMPONENTSPATH)/lib/httpclient-4.3.6.jar:$(HTTPCOMPONENTSPATH)/lib/httpclient-cache-4.3.6.jar:$(HTTPCOMPONENTSPATH)/lib/httpcore-4.3.3.jar:$(HTTPCOMPONENTSPATH)/lib/httpmime-4.3.6.jar:$(KAFKAPATH)/project/boot/scala-2.8.0/lib/scala-compiler.jar:$(KAFKAPATH)/project/boot/scala-2.8.0/lib/scala-library.jar:$(KAFKAPATH)/core/target/scala_2.8.0/kafka-0.7.2.jar:$(KAFKAPATH)/core/lib/*.jar:$(KAFKAPATH)/perf/target/scala_2.8.0/kafka-perf-0.7.2.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/jopt-simple-3.2.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/log4j-1.2.15.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/snappy-java-1.0.4.1.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/zkclient-0.1.jar:$(KAFKAPATH)/core/lib_managed/scala_2.8.0/compile/zookeeper-3.3.4.jar:./jewelcli-0.8.7.jar:./java-statsd-client-3.1.0.jar:./kafkaSpraynozzle.jar"
```

Comment:
I've seriously got to figure out how to convert this to maven. It's getting complex enough that it's warranted. I just wish the kafka guys would put kafka 0.7.2 in maven central...